packages/tools/src/user-interaction.ts

```diff
@@ -0,0 +1,88 @@
+import { confirm, select, text } from '@clack/prompts'
+import { tool } from 'ai'
+import { z } from 'zod'
+
+type Spinner = {
+  start: (msg?: string) => void
+  stop: (msg?: string, code?: number) => void
+  message: (msg?: string) => void
+}
+
+export const makeInteractiveToolset = (s: Spinner) => {
+  return {
+    askQuestion: askQuestion(s),
+    getConfirmation: getConfirmation(s),
+    presentOptions: presentOptions(s),
+    concludeTask,
+  }
+}
+
+export const askQuestion = (s: Spinner) =>
+  tool({
+    description: 'Interact with user to as him a question',
```

Comment:
nit: remove "interact with user to" in favor of "ask user a question". Also, there's a typo - should be "ask" instead of "as"